packages/TextEditor/src/TextEditor.js

```diff
@@ -1,5 +1,79 @@
 import React from "react";
+import PropTypes from "prop-types";
+import { Editor } from "@tinymce/tinymce-react";
+import { kind as kindType } from "./types";
+import * as simpleConfig from "./config/simple.config";
 
-export default function TextEditor() {
-  return <>Text editor</>;
+function getPlugins(kind) {
+  if (kind === kindType.SIMPLE) {
+    return simpleConfig.plugins;
+  }
 }
+
+function getToolbar(paramKind) {
+  if (paramKind === kindType.SIMPLE) {
+    return simpleConfig.toolbar;
+  }
+}
+
+export default function TextEditor(props) {
+  const { height, isDisabled, apiKey, onChange, value, defaultValue, kind } = props;
+
+  if (value && defaultValue) {
+    throw Error("The component can only accept a value or a defaultValue prop, not both");
+  }
+
+  if (!value && !defaultValue) {
```

Comment:
This will fail if `value` and/or `defaultValue` is an empty string.

Maybe we should check these values for being a non-empty strings instead. That way we will also check for the other invalid types, such as objects, numbers, or arrays

@nahumzs

Reply:
I will add this in my incoming PR thank you @AndreyChernykh